metashare/api/models.py

```diff
@@ -216,6 +223,11 @@ def get_a_matching_user(self):
 
         return None
 
+    def queue_populate_github_users(self):
```

Comment:
It doesn't look like this is called from anywhere? When are we planning to populate this data? Shouldn't it probably happen along with the `get_repo_id` mixin?

Reply:
The job logic gets called from the management command, but this doesn't get called anywhere, correct. I wasn't sure what might trigger it!

Reply:
Right -- it's the kind of thing that should happen at least once initially for every repo, but then probably provide a way to trigger a re-check?